tests/cron.test.ts

```diff
@@ -3,208 +3,190 @@ import sinon from 'sinon';
 import { CronJob, CronTime } from '../src';
 
 describe('cron', () => {
-	// eslint-disable-next-line jest/no-standalone-expect
-	afterEach(() => expect.hasAssertions());
+	let callback: jest.Mock;
+
+	beforeEach(() => {
+		callback = jest.fn();
+	});
+
+	afterAll(() => {
+		jest.clearAllMocks();
+	});
+
+	afterEach(() => sinon.restore());
```

Comment:
Looks like you've removed the `expect.hasAssertions()` from the `afterEach` hook. Please add it back as it's essential to how our tests work.

Reply:
I reverted that as well when rolling back the done callback.